### PR TITLE
Update 2023-04-12-grafana-tutorial.mdx

### PR DESCRIPTION
**Pull request by tris0laris**

### CODE REVIEW

File: blog/2023-04-12-grafana-tutorial.mdx [hunk 1]
```diff
@@ -376,3 +376,7 @@ insights into how weather conditions and taxi fares affect each other.
 
 If you like this content and want to see more tutorials about third-party
 integrations, let us know [in our Slack Community]({@slackUrl@}).
+
+## Additional Resources
+
+[What is Time-Series Data?](https://questdb.io/blog/time-series-data/)
```
Comment (argshook) [inline blog/2023-04-12-grafana-tutorial.mdx:382]:
```suggestion
[What is Time-Series Data?](/blog/time-series-data/)
```

Comment (tris0laris) [reply blog/2023-04-12-grafana-tutorial.mdx:382]: good to know

Comment (tris0laris) [reply blog/2023-04-12-grafana-tutorial.mdx:382]: @argshook i may also have done this PR from a preview Netlify page state. Lmk if I should re-do this PR

Comment (argshook) [reply blog/2023-04-12-grafana-tutorial.mdx:382]: > i may also have done this PR from a preview Netlify page state. Lmk if I should re-do this PR

nono, all good, no need to re-do

**Review by argshook — APPROVED** [review]

**Pull request merged**